src/storage/src/store.rs

```diff
@@ -309,13 +312,29 @@ pub trait LocalStateStore: StaticSendSync {
     ) -> Self::MayExistFuture<'_>;
 }
 
+/// If `exhaust_iter` is true, prefetch will be enabled. Prefetching may increase the memory
+/// footprint of the CN process because the prefetched blocks cannot be evicted.
+#[derive(Default, Clone, Copy)]
+pub struct PrefetchOptions {
+    /// `exhaust_iter` is set `true` only if the return value of `iter()` will definitely be
+    /// exhausted, i.e., will iterate until end.
+    pub exhaust_iter: bool,
+}
+
+impl PrefetchOptions {
+    pub fn new_for_exhaust_iter(exhaust_iter: bool) -> Self {
```

Comment:
One more thing: we don't need parameter for this method...

Reply:
> One more thing: we don't need parameter for this method...

fixed